Formula/c/cmctl.rb

```diff
@@ -40,7 +39,7 @@ def install
     assert_match "cmctl", shell_output("#{bin}/cmctl help")
     # We can't make a Kubernetes cluster in test, so we check that when we use a remote command
     # we find the error about connecting
-    assert_match "error: error finding the scope of the object", shell_output("#{bin}/cmctl check api 2>&1", 1)
+    assert_match "error: error finding the scope of the object", shell_output("#{bin}/cmctl check api 2>&1", 129)
```

Comment:
This is a bug that was introduced in https://github.com/cert-manager/cmctl/pull/33 and will be fixed in https://github.com/cert-manager/cmctl/pull/40.